src/test/resources/application.yml

```diff
@@ -2,12 +2,21 @@ api:
   key: some-api-key
   url: http://api
 
+aws:
+  accessKeyId: some-access-key
+  region: eu-west-2
+  secretAccessKey: some-secret-key
+
 barcodeGenerator:
   host: http://barcode-generator
 
 cdn:
   host: http://cdn
 
+chips:
```

Comment:
You will need to extend this too to include same test values for your new env vars